besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -2506,6 +2508,17 @@ && rpcWsAuthenticationCredentialsFile() == null
   }
 
   private ApiConfiguration apiConfiguration() {
+    if (!apiPriorityFeeLimitingEnabled
```

Comment:
There is a standard way to do this check, that also works when someone passed the default value, using `CommandLineUtils::checkOptionDependencies` see how it is already used for examples. Then having a check that the lower bound is <= the upper bound will be nice too.

Reply:
done